docs/api-reference/carto/heatmap-tile-layer.md

```diff
@@ -92,6 +92,11 @@ When `colorDomain` is specified, a pixel with `minValue` is assigned the first c
 
 The color palette used in the heatmap, as an array of colors [color1, color2, ...]. Each color is in the format of `[r, g, b]`. Each channel is a number between 0-255.
 
+#### `intensity` (number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square") {#intensity}
+
+* Default: `1`
+
+Value that is multiplied with the total weight at a pixel to obtain the final weight. A value larger than `1` biases the output color towards the higher end of the spectrum, and a value less than `1` biases the output color towards the lower end of the spectrum.
```

Comment:
I chose the unit area to be the whole world (in Mercator) as it is at least non-ambiguous. The downside is that it leads to large values of density. I chose not to use square meters (or km) as this would then need to be projected and wouldn't correspond to the partitioning of quadbins.

With the above choice you can think of density (when `getWeight` is just returning a count) as the total count across the world if all cells worldwide had the same density.